salt/pillar/git_pillar.py

```diff
@@ -350,6 +350,21 @@
         - __env__ https://mydomain.tld/pillar-appdata.git:
           - mountpoint: web/server/
 
+.. _git_pillar_update_interval:
+
+git_pillar_update_interval
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. versionadded:: 2020.2.0
```

Comment:
This should be `Neon`

Reply:
Done (also in `doc/ref/configuration/master.rst`), rebased and force-pushed.